server/test/api/patients.js

```diff
@@ -56,8 +56,7 @@ describe('The /patients API', function () {
   var mockMissingRequest = {
     finance : {
       debitor_group_uuid : '4de0fe47-177f-4d30-b95f-cff8166400b4'
-    },
-    medical : missingParamsPatient
```

Comment:
If the test does not pass, it means the test found a bug.  Why did changes to exchange rate break the patients API?  All tests pass on master.